src/panels/energy/ha-panel-energy.ts

```diff
@@ -71,7 +74,23 @@ class PanelEnergy extends LitElement {
           <hui-energy-period-selector
             .hass=${this.hass}
             collectionKey="energy_dashboard"
-          ></hui-energy-period-selector>
+          >
+            ${this.hass.user?.is_admin
+              ? html`
+                  <ha-list-item
+                    slot="overflow-menu"
+                    graphic="icon"
+                    @request-selected=${this._navigateConfig}
+                  >
+                    <ha-svg-icon slot="graphic" .path=${mdiPencil}>
+                    </ha-svg-icon>
+                    ${this.hass!.localize(
+                      "ui.panel.lovelace.menu.configure_ui"
```

Comment:
I would make this text different, `Energy settings` or something? As it is also configuring costs etc.

Reply:
The page title from the quick-bar is "Energy Configuration". Could use that to be consistent, or should we keep it as an imperative statement like the other overflow menu items? Maybe "Configure Energy"? (is Energy a proper noun to be capitalized in this context?)

Anyway I don't care too much, so just let me know whatever you'd like to see and I'll put it in.

Reply:
Yeah, `Energy Configuration` sounds great!

Reply:
updated to `Energy Configuration`